core/src/main/java/org/axonframework/eventsourcing/eventstore/jdbc/JdbcEventStorageEngine.java

```diff
@@ -78,148 +92,55 @@
-             null, null);
+    protected JdbcEventStorageEngine(Builder builder) {
+        super(builder);
+        this.connectionProvider = builder.connectionProvider;
+        this.transactionManager = builder.transactionManager;
+        this.dataType = builder.dataType;
+        this.schema = builder.schema;
+        this.lowestGlobalSequence = builder.lowestGlobalSequence;
+        this.maxGapOffset = builder.maxGapOffset;
+        this.gapTimeout = builder.gapTimeout;
+        this.gapCleaningThreshold = builder.gapCleaningThreshold;
     }
 
     /**
-     * Initializes an EventStorageEngine that uses JDBC to store and load events using the default {@link EventSchema}.
-     * The payload and metadata of events is stored as a serialized blob of bytes using the given {@code serializer}.
+     * Instantiate a Builder to be able to create a {@link JdbcEventStorageEngine}.
      * <p>
-     * Events are read in batches of 100. The given {@code upcasterChain} is used to upcast events before
-     * deserialization. The same {@link org.axonframework.serialization.Serializer} is used for both snapshots and
-     * events.
-     *
-     * @param serializer                   Used to serialize and deserialize event payload and metadata, and snapshots.
-     * @param upcasterChain                Allows older revisions of serialized objects to be deserialized.
-     * @param persistenceExceptionResolver Detects concurrency exceptions from the backing database. If {@code null}
-     *                                     persistence exceptions are not explicitly resolved.
-     * @param connectionProvider           The provider of connections to the underlying database.
-     * @param transactionManager           The instance managing transactions around fetching event data. Required by
-     *                                     certain databases for reading blob data.
-     */
-    public JdbcEventStorageEngine(Serializer serializer, EventUpcaster upcasterChain,
-                                  PersistenceExceptionResolver persistenceExceptionResolver,
-                                  ConnectionProvider connectionProvider, TransactionManager transactionManager) {
-        this(serializer, upcasterChain, persistenceExceptionResolver, serializer, connectionProvider,
-             transactionManager);
-    }
-
-    /**
-     * Initializes an EventStorageEngine that uses JDBC to store and load events using the default {@link EventSchema}.
-     * The payload and metadata of events is stored as a serialized blob of bytes using the given {@code serializer}.
+     * The following configurable fields have defaults:
+     * <ul>
+     * <li>The snapshot {@link Serializer} defaults to {@link org.axonframework.serialization.xml.XStreamSerializer}.</li>
+     * <li>The {@link EventUpcaster} defaults to an {@link org.axonframework.serialization.upcasting.event.NoOpEventUpcaster}.</li>
```

Comment:
Yes I do, because this class does not directly use the `NoOpEventUpcaster`. It's the Builder which this Builder implements which does that. 
I wanted to make these defaults clear in the implementations of Builders too, hence why I copied it over. This does require the long imports however.